it/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -62,42 +62,42 @@
 <!ENTITY showConnections.label         "Connessione...">
 <!ENTITY showConnections.accesskey     "n">
 
-<!ENTITY offlineCompact.label "Compatta le cartelle solo se si guadagnano più di ">
+<!ENTITY offlineCompact.label "Comprimi le cartelle se lo spazio risparmiato è maggiore di ">
 <!ENTITY offlineCompact.accesskey "C">
 <!ENTITY kb.label "kB">
 
 <!-- Certificates -->
 <!ENTITY certSelection.label "Quando un server richiede il mio certificato:">
-<!ENTITY certs.auto                      "Selezionane uno automaticamente">
+<!ENTITY certs.auto                      "Seleziona certificato automaticamente">
 <!ENTITY certs.ask                       "Chiedi ogni volta">
 
-<!ENTITY viewCertificates.label         "Mostra certificati">
+<!ENTITY viewCertificates.label         "visualizza certificati">
```

Comment:
CORRECTIONS NEEDED. A capital letter is needed at the beginning of a sentence. It should be as follows: 

+<!ENTITY viewCertificates.label         "Visualizza certificati">